frontend/DashboardPage.tsx

```diff
@@ -0,0 +1,177 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Avatar, Typography } from '@material-ui/core';
+import MailIcon from '@material-ui/icons/Mail';
+import VideoCallIcon from '@material-ui/icons/VideoCall';
+import SearchIcon from '@material-ui/icons/Search';
+import EditIcon from '@material-ui/icons/Edit';
+import InfoIcon from '@material-ui/icons/Info';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+export type MailProps ={
+    profilePic: string;
+    name: string;
+}
+
+type DashboardItems = {
+    icon: JSX.Element;
+    optionsName: string;
+}
+
+type Maillist = {
+    profile: JSX.Element;
+    name: string;
+    message: string;
+    dateReceived: string;
+
+}
+
+const useStyles = makeStyles({
+    bar: {
+        cursor: 'pointer',
+        padding: 10,
+        borderBottom: "2px",
+        borderBottomColor: "black",
+    },
+
+    barName: {
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+    },
+    mainbar: {
+        cursor: 'pointer',
+        padding: 10,
+    },
+    main:{
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+    },
+    mailSection:{
+        flexGrow: 1,
+    }
+
+   
+
+});
+
+
+export default function DashboardPage() {
+    const classes = useStyles();
+
+    // Add the options for the sidebar to the Dashboard list.
+    var options: DashboardItems[] = [
+        {
+            icon: <MailIcon />,
+            optionsName: 'Penpal Mails',
+        },
+        {
+            icon: <VideoCallIcon />,
+            optionsName: 'Start Video Call',
+        },
+        {
+            icon: <SearchIcon />,
+            optionsName: 'Find Penpal',
+        },
+        {
+            icon: <EditIcon />,
+            optionsName: 'Edit Profile',
+        },
+        {
+            icon: <InfoIcon />,
+            optionsName: 'Tutorials',
+        },
+    ];
+
+    var contents: Maillist[] = [
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe',
+            message: 'New Video',
+            dateReceived: 'Oct 22',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe Jr.',
+            message: 'New Video',
+            dateReceived: 'Oct 21',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'Thomas Jones',
+            message: 'New Video',
+            dateReceived: 'Oct 20',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe Jr',
+            message: 'New Video',
+            dateReceived: 'Oct 19',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe',
+            message: 'New Video',
+            dateReceived: 'Oct 18',
+        },
+        
+    ];
+
+
+    return (
+        
+    <>
+        <div style={{ margin: 0, padding: 0, boxSizing: 'border-box'}}>

```

Comment:
Let's avoid inline styling as much as possible. Sometimes it is unavoidable, such as with external libraries. Also, you should be using a MUI Box here.